LanguageFeatures/Augmentation-libraries/defining_augmentation_A01_t01.dart

```diff
@@ -2,11 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion Declarations in any library augmentation or the augmented library
-/// are visible to all of the others, including private ones.
+/// @assertion Augmentation declarations interact with part files mainly in
+/// restrictions on where an augmenting declaration may occur relative to the
+/// declaration it augments.
 ///
 /// @description Checks that declarations in an augmentation are visible in a
```

Comment:
I can't help asking what this means. The word 'augmentation' could refer to a Dart file, but they would be called 'part files' now, rather than 'augmentation libraries' or anything like that. So maybe it should be adjusted like this?:
```suggestion
/// @description Checks that declarations in a part are visible in the
```

Reply:
May be "Checks that declarations in an augmenting part file are visible in the main library."?